src/main.rs

```diff
@@ -21,6 +21,26 @@ pub struct Args {
   #[arg(short, long)]
   width: usize,
 
+  /// Separator between tabs.
+  #[arg(long, value_name = "FACE")]
+  separator: Option<String>,
+
+  /// Face to use on focused tabs.
+  #[arg(long, value_name = "FACE")]
+  focusedface: Option<String>,
+
+  /// Face to use on inactive tabs.
+  #[arg(long, value_name = "FACE")]
+  inactiveface: Option<String>,
+
+  /// Face to use on separators.
+  #[arg(long, value_name = "FACE")]
+  defaultface: Option<String>,
+
+  /// Face to use on a modified tab indicator.
+  #[arg(long, value_name = "FACE")]
+  modifiedface: Option<String>,
```

Comment:
Let's do something like this instead so we don't need `unwrap_or` later:
```suggestion
  /// Separator between tabs.
  #[arg(long, value_name = "FACE", default_value_t="|")]
  separator: String,

  /// Face to use on focused tabs.
  #[arg(long, value_name = "FACE", default_value_t="Prompt")]
  focused_face: String,

  /// Face to use on inactive tabs.
  #[arg(long, value_name = "FACE", default_value_t="LineNumbers")]
  inactive_face: String,

  /// Face to use on separators.
  #[arg(long, value_name = "FACE", default_value_t="Default")]
  default_face: String,

  /// Face to use on a modified tab indicator.
  #[arg(long, value_name = "FACE", default_value_t="DiagnosticError")]
  modified_face: String,
```

Reply:
my bad. i couldn't figure out how to add default values here